src/main/java/com/celements/course/service/CourseConfirmState.java

```diff
@@ -0,0 +1,25 @@
+package com.celements.course.service;
+
+import javax.validation.constraints.NotNull;
+
+import com.google.common.base.Optional;
+
+public enum CourseConfirmState {
```

Comment:
why is this class here?

Reply:
OK, i think it just went in after a merge some days ago. i will delete it again.